app/javascript/packs/step-2/nextStepButton.vue.erb

```diff
@@ -0,0 +1,42 @@
+<template>
+  <button class="ui teal right labeled icon button right floated"
+  v-bind:class="{ loading: this.isRequestInProgress, disabled: this.isRequestInProgress }"
+  @click="nextStepButtonClicked">
+    <label><%= I18n.t('next_step') %></label>
+    <i class="arrow right icon"></i>
+  </button>
+</template>
+
+<script>
+    import { mapState, mapMutations, mapActions } from 'vuex'
+    import * as types from './store/mutationTypes'
+
+    export default {
+        name: 'next-step-button',
+        computed: {
+            ...mapState({
+                diagnosisContent: state => state.step2Store.diagnosisContent,
+                isRequestInProgress: state => state.step2Store.isDiagnosisRequestUnderWay
```

Comment:
I don't find `isDiagnosisRequestUnderWay` anywhere else in this PR. It is elsewhere in the project?